modules/sdk-lib-mpc/test/unit/tss/ecdsa/dklsDsg.ts

```diff
@@ -0,0 +1,63 @@
+import { DklsDsg } from '../../../../src/tss/ecdsa-dkls';
+import * as fs from 'fs';
+import * as crypto from 'crypto';
+import should from 'should';
+
+describe('DKLS Dsg 2x3', function () {
+  it(`should create signatures`, async function () {
```

Comment:
It would be nice to have a `xit` block that creates the fixtures in case we need to regenerate them. 

Or add a comment block with instructions on how to generate the fixtures.

Reply:
fixed (added comment)